winui/Ribbon/Ribbon-QAT.md

```diff
@@ -121,6 +122,22 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
 
 ![WinUI Ribbon QAT menu items](Ribbon-QAT-images/winui-ribbon-qat-menu-items.png)
 
+## Adding items using QAT Customize dialog window
+
+To open QAT customize dialog window, select `MoreCommands` option from the ContextMenu of the QAT. This dialog window showcases all available commands within the Ribbon control.
+
+By using `Choose commands from` option, you can filter the command based on ribbon tabs. Then, select the desired command and click `Add` button to add the command to the right Pane of the QAT dialog window. Finally, click `OK`.
```

Comment:
Filter the command based on ribbon tabs by using the `Choose commands from` option. Then, select the desired command and click `Add` to add the command to the right Pane of the QAT dialog window. Finally, click `OK`.

Reply:
modified